website/teacher.py

```diff
@@ -473,7 +473,7 @@ def update_adventure(user):
                 return gettext('adventure_duplicate'), 400
 
         try:
-            content = body['content'].format(**hedy_content.KEYWORDS.get(g.keyword_lang))
+            body['content'].format(**hedy_content.KEYWORDS.get(g.keyword_lang))
```

Comment:
maybe add a comment here why this check is needed? This looks a bit magical :)

Reply:
Added!